controlplane/kubeadm/main.go

```diff
@@ -44,11 +44,13 @@ import (
 	kcpv1alpha4 "sigs.k8s.io/cluster-api/controlplane/kubeadm/api/v1alpha4"
 	kcpv1 "sigs.k8s.io/cluster-api/controlplane/kubeadm/api/v1beta1"
 	kubeadmcontrolplanecontrollers "sigs.k8s.io/cluster-api/controlplane/kubeadm/controllers"
+	kcpwebhooks "sigs.k8s.io/cluster-api/controlplane/kubeadm/webhooks"
```

Comment:
```suggestion
	"sigs.k8s.io/cluster-api/controlplane/kubeadm/webhooks"
```
I think it's fine to just use webhooks here, there is no need to distinguish like for the APIs

Reply:
It was aliased to differentiate it from the`"sigs.k8s.io/controller-runtime/pkg/webhook"`  package.

Reply:
Ah got it, I just checked for a conflict :)

Up to you if you want to keep it or change it